### PR TITLE
chore: modify rust openapi gen script for extensions

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Allows it to be run from parent git module.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Review by chriswldenyer — APPROVED** [review]

File: scripts/rust/generate-openapi-bindings.sh [hunk 1]
```diff
@@ -117,7 +119,7 @@ if [[ "$skip_if_md5_same" = "yes" ]]; then
 fi
 
 # Cleanup the existing autogenerated code
-git clean -f -X "$TARGET" || true
+( cd "$TARGET"; git clean -f -X . )
```
Comment (blaisedias) [inline scripts/rust/generate-openapi-bindings.sh]: for consistency shouldn't this be `$REAL_TARGET`?

**Review by blaisedias — APPROVED** [review]
Looks plausible

**Comment by tiagolobocastro** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**